<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlueToothItem</class>
 <widget class="QWidget" name="BlueToothItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>66666</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LB_NAME">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LB_ADDR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_CONNECT">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
